ui/admin/app/templates/scopes/scope.hbs

```diff
@@ -33,218 +33,314 @@
   />
 {{/if}}
 
-<Rose::Layout::Sidebar as |sidebarLayout|>
-
-  {{#if this.session.isAuthenticated}}
-    <sidebarLayout.sidebar>
-
-      {{#if @model.isGlobal}}
-        <Rose::Nav::Sidebar @title={{t 'titles.general'}} as |nav|>
-          <nav.link @route='scopes.scope.scopes'>
-            <Rose::Icon @name='flight-icons/svg/org-16' @size='medium' />
-            {{t 'resources.org.title_plural'}}
-          </nav.link>
-
-          {{#if (can 'navigate scope' @model collection='session-recordings')}}
-            <nav.link @route='scopes.scope.session-recordings'>
-              <Rose::Icon @name='flight-icons/svg/film-16' @size='medium' />
-              {{t 'resources.session-recording.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if (can 'navigate scope' @model collection='storage-buckets')}}
-            <nav.link @route='scopes.scope.storage-buckets'>
-              <Rose::Icon
-                @name='flight-icons/svg/cloud-upload-16'
-                @size='medium'
-              />
-              {{t 'resources.storage-bucket.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if (can 'navigate scope' @model collection='policies')}}
-            <nav.link @route='scopes.scope.policies'>
-              <Rose::Icon
-                @name='flight-icons/svg/hourglass-16'
-                @size='medium'
-              />
-              {{t 'resources.policy.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if (can 'navigate scope' @model collection='aliases')}}
-            <nav.link @route='scopes.scope.aliases'>
-              <Rose::Icon @name='flight-icons/svg/at-sign-16' @size='medium' />
-              {{t 'resources.alias.title_plural'}}
-            </nav.link>
-          {{/if}}
-          {{#if
-            (and
-              (can 'navigate worker' @model collection='workers')
-              (feature-flag 'byow')
-            )
-          }}
-            <nav.link @route='scopes.scope.workers'>
-              <Rose::Icon @name='flight-icons/svg/cpu-16' @size='medium' />
-              {{t 'titles.workers'}}
-            </nav.link>
-          {{/if}}
-        </Rose::Nav::Sidebar>
+{{#if this.session.isAuthenticated}}
+  {{#in-element this.sideBarContainer}}
```

Comment:
Why did we have to portal this in?

Reply:
thats a good point, i think the same thing can be accomplished using the side nav list directly.

Reply:
Edit: Just realized you are referring to the `in-element` helper usage. That is because we don't show the side nav on all routes but we do want the side nav to be inserted into the correct hds::frame  component.

Reply:
My 2 cents tip: add a comment in the `ui/admin/app/templates/application.hbs` file, above the `<Frame.Sidebar` declaration, to explain what is going on. I wasn't sure how you were adding the content to the sidebar, until I read this comment above.

Reply:
I don't know if/how it would apply to your use case, but have a look at how Cloud UI has solved the problem of having some pages without a sidebar: https://github.com/hashicorp/cloud-ui/blob/main/addons/core/package/src/components/hcp-app-frame.gts#L148

(essentially, if you pass `@hasSidebar={{a false value}}` the sidebar is not rendered at all)

This may avoid doing double portalling.

Reply:
interesting, thank you @didoo for the link + info! ill try this out since i think that will work for this use case as well!